src/svg/boxPlot.js

```diff
@@ -0,0 +1,151 @@
+import d3 from 'd3';
+
+// Renders a box plot series as an SVG path based on the given array of datapoints.
+export default function() {
+
+    var x = function(d, i) { return d.x; },
+        y = function(d, i) { return d.y; },
+        boxHigh = function(d, i) { return d.boxHigh; },
+        boxLow = function(d, i) { return d.boxLow; },
+        whiskerHigh = function(d, i) { return d.whiskerHigh; },
+        whiskerLow = function(d, i) { return d.whiskerLow; },
+        orient = 'vertical',
+        barWidth = d3.functor(5);
+
+    var boxPlot = function(data) {
+
+        return data.map(function(d, i) {
+            var halfWidth = barWidth(d, i) / 2,
+                whiskerTotal = whiskerHigh(d, i) - whiskerLow(d, i),
+                whiskerToBotBox = boxLow(d, i) - whiskerLow(d, i),
+                boxToMid = y(d, i) - boxLow(d, i),
+                midToBox = boxHigh(d, i) - y(d, i),
+                boxToWhiskerHigh = whiskerHigh(d, i) - boxHigh(d, i),
+                yBottom = y(d, i) - whiskerLow(d, i),
+                yTop = whiskerHigh(d, i) - y(d, i),
+                xBottom = x(d, i) - whiskerLow(d, i),
+                xTop = whiskerHigh(d, i) - x(d, i);
+
+            var boxSeriesVertical = '';
```

Comment:
considering the if / else block below - is there any reason why you have separate `boxSeriesVertical` and `boxSeriesHorizontal` variables? wouldn't a single `boxSeriesPath` variable be sufficient?